db/migrate/20160114200145_add_mirror_on_sync_to_repositories.rb

```diff
@@ -0,0 +1,6 @@
+class AddMirrorOnSyncToRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :mirror_on_sync, :boolean, :default => true, :null => false
+    update "UPDATE katello_repositories SET mirror_on_sync = 'f'"
```

Comment:
@ehelms @jlsherrill  Idea was that we do not want to force mirroring for existing repositories. Basically leave the existing repos as they are.  But we want to set mirroring as default for new repositories.

If we want mirroring  for existing we'd have to force sync in addition to setting the mirroring flag on.